AdminWebsite/AdminWebsite/ClientApp/src/app/booking/services/participant-mapper.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
```

Comment:
Let's talk about this mapper, I'm not sure it's needed.

Reply:
Yeah seems overkill. I was thinking to move these into PaticipantModel. Does that make sense to you?

Reply:
Removed the service but I kept the code as static methods in ParticipantModel.